docs/sphinx/helpers.rst

```diff
@@ -5,6 +5,15 @@ Helpers
 
 Collection of simple helper functions that abstract some specifics of the raw API.
 
+Connecting
+----------
+
+.. code-block:: python
+
+    from elasticsearch import Elasticsearch, helpers
```

Comment:
Any reason to import helpers here? Do we use it anywhere?

Reply:
We don't; I just wanted to have at least one example of importing the rest of them, other than bulk?
But we can also skip this one

Reply:
Yes it seems weird if we don't use. Either we call `helpers.bulk` or we drop this IMO.

Reply:
fixed